### ISSUE TITLE
DB Connection lost when working with 2 databases

### ISSUE THREAD

**Issue by pstorch** [keepassxreboot/keepassxc-browser#1118]
I work with 2 KeePassXC databases. 1 for work 1 for private. Sometimes I need both open. I connect Firefox to both, but only the last one is remembered by Firefox. When I reopen the KeePassXC-Browser settings page again, the first connected database is gone.

## Expected Behavior
Both DB connections should be kept

## Current Behavior
Only last connected DB is kept

## Steps to Reproduce (for bugs)
1. Open DB 1 in KeePassXC
2. Open DB 2 in KeePassXC
3. Connect DB1 in Firefox KeePassXC-Browser extension
4. Connect DB2 in Firefox KeePassXC-Browser extension
5. Both DBs are connected now
6. Close the Settings page
7. Reopen the Settings page
8. Connection to DB 1 is gone

## Debug info
KeePassXC - 2.6.2
KeePassXC-Browser - 1.7.3
Operating system: Win32
Browser: Mozilla Firefox 82.0

**Comment by varjolintu** [comment]
Does it matter if you connect the extension through the popup and not from the Settings page?

**Comment by pstorch** [comment]
It's the same through the popup.

**Comment by droidmonkey** [comment]
Enable search all open databases in the KeePassXC browser settings

**Comment by varjolintu** [comment]
Both DB's must be connected to KeePassXC for that option to work.

**Comment by pstorch** [comment]
> Enable search all open databases in the KeePassXC browser settings and only connect one db

Then the extension doesn't find the entries from the other DB.

**Comment by varjolintu** [comment]
One thing you could try is to create a new temporary profile and see if the bug happens there also.

**Comment by pstorch** [comment]
I tried it with a new firefox profile, same behaviour.

**Comment by varjolintu** [comment]
I cannot reproduce this.

**Comment by pstorch** [comment]
I think I found the cause. In the past I only had 1 combined DB. I splited the DB by making a (file)copy and removed the entries in both which don't belong there.

I found this option to renew the ID of the DB root group:

![image](https://user-images.githubusercontent.com/5598124/99548188-a8581c00-29b8-11eb-9907-cc7bfc1ef526.png)

Then I connected the databases again. Now I can work with both in parallel and both stay connected.
Thanks for your effort.
